core/templates/css/oppia.css

```diff
@@ -458,6 +458,10 @@ h3 {
   margin: 18px 0 18px 0;
 }
 
+.oppia-contact-mail {
+  color: #052b6c;
+}
```

Comment:
@DubeySandeep the color contrast is not upto the mark as per the W3 aa standards that is causing the axeDevtools test to fail.

Reply:
Thanks for the explanation, @Patel-Muhammad ! Can you please check why `#0844aa`  [oppia.css, line 465] works for all other places and not here? Also, do we use `#052b6c` for links in other places in the codebase?

Reply:
I am not sure if we should change the color at all places in the codebase, there may be the situation where the font size of the link at other places is large or the background is different so the test may not fail there. Although I believe that it is a thing that can be thought out properly then be implemented.

Reply:
I'm not asking to add a new color to all the links, instead, I'm worried about managing two different colors for links.  (Maybe we are already have different colors for link)

> Can you please check why #0844aa [oppia.css, line 465] works for all other places and not here? Also, do we use #052b6c for links in other places in the codebase?

Answers to these^ questions will definitely help us resolve this thread!

Reply:
@DubeySandeep I didnt find any other links that are using that color in the codebase, mostly the anchor tags are inside buttons and having different background. Let me know if I am missing something.

Reply:
@Patel-Muhammad Is it possible to move this style to the related contact page css file?

Reply:
@DubeySandeep has a good point, we do need to verify consistency here. This is a green link on white background right? If so we should ensure that those links are represented the same way throughout the codebase.

@Patel-Muhammad to be clear, are you saying that (a) there are no other green links on white backgrounds throughout the site, or (b) there are green links on white backgrounds elsewhere but there is a specific reason to use a different colour here? Please clarify and include screenshots if needed. We should have one set of colours for this use case throughout the whole codebase.

Reply:
Hi @DubeySandeep there is no link in oppia where there is green link over the white background

Reply:
Actually, no style is overriding it, the style is not applying on the text in the first place.

Reply:
is that expected? Also, do we know why it's not working from the component style page but working fine from the global css page? (Asking this as we are expecting this style to be written in the component css)

Reply:
@Patel-Muhammad I assume you assigned me to this PR because of this comment thread ... but I have the same comment as Sandeep. What's the actual issue here?